__tests__/data/moengage_output.json

```diff
@@ -0,0 +1,343 @@
+[
```

Comment:
Better to use 2-space indentation to keep it similar to others

Reply:
done